discord-bot/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "osdc-bot",
+  "version": "1.0.0",
+  "description": "A simple bot for OSDC Discord Server",
+  "main": "bot.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Add a script for `start` which executes `node bot.js`. And update the README accordingly.